book/src/assets.md

```diff
@@ -0,0 +1,13 @@
+# Assets
+
+**Assets** are configuration that are loaded by a game when it is run. These may be textures, sounds, game level scripts, and so on. In fact, any non-player-specific information loaded by the executable can be called an asset. These are usually stored as files, and distributed alongside the game.
```

Comment:
Instead of "Assets are configuration that are..." maybe, "Assets are data that's loaded by a game when it is run."

Reply:
+1